modules/loki-entry-out-of-order-errors.adoc

```diff
@@ -0,0 +1,75 @@
+// Module is included in the following assemblies:
+// cluster-logging-loki.adoc
+
+:_content-type: PROCEDURE
+[id="loki-entry-out-of-order-messages_{context}"]
+= Troubleshooting Loki entry out of order errors
+
+If your Fluentd instance forwards a large block of messages that exceeds the rate limit to a Loki logging system, Loki generates `entry out of order` errors. You can fix this issue by modifying the `LokiStack` custom resource (CR).
```

Comment:
Instead of 'fluentd', we can mention collector. Since we have defined collector as vector in the ClusterLogging CR, it doesn't make sense to have fluentd here.

Reply:
Presumably it needs to be changed anywhere in the docs then that Fluentd is mentioned? I think there are other instances.

Reply:
I will maybe for this PR just update this instance, since we also have https://issues.redhat.com/browse/OBSDOCS-460 which I think will cover any other additional required changes